Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -414,7 +420,9 @@ private void EmitFillOrderEvent(Messages.Matched message, Symbol symbol, GDAXFil
             var fillPrice = message.Price;
             var fillQuantity = direction == OrderDirection.Sell ? -message.Size : message.Size;
             var isMaker = order.BrokerId[0] == message.MakerOrderId;
-            var orderFee = GetFillFee(symbol, fillPrice, fillQuantity, isMaker);
+            var orderFee = new OrderFee(new CashAmount(
+                GetFillFee(symbol, fillPrice, fillQuantity, isMaker),
+                symbol.Value.Substring(3)));
```

Comment:
This line assumes the _base_ currency has a fixed length of 3 characters. 
Since Coinbase is incrementally adding new symbols with 4-char base currencies and 4-char quote currencies (e.g. `MANA/USDC`, `LOOM/USDC`), I think it might be worth to move this logic to private methods such as `GetBaseCurrency(string ticker)` and `GetQuoteCurrency(string ticker)`.

For now we might just use a hardcoded list of available quote currencies (USD, EUR, GBP, BTC, USDC), in the future we could fetch this information directly from the brokerage at startup time.

Reply:
Good point! 
After digging a bit more on this I decided to use `order.PriceCurrency` or, if empty, `_algorithm.Securities[symbol].SymbolProperties.QuoteCurrency` (as done in other brokers implementation).
Regarding the fixed 3 char lenght, it seems to be scattered among othe places too e.g. https://github.com/QuantConnect/Lean/blob/master/Common/Securities/SecurityService.cs#L66 https://github.com/QuantConnect/Lean/blob/master/Common/Securities/Cash.cs#L90 which would require more changes, so I'm thinking of openning a new issue to address it separatly.

Reply:
New issue at https://github.com/QuantConnect/Lean/issues/2744